src/views/ActivityAppFeed.vue

```diff
@@ -222,7 +222,7 @@ watch(props, () => {
 		font-size: 20px;
 		line-height: 44px; // to align height with the app navigation toggle
 		// Align with app navigation toggle
-		margin: var(--app-navigation-padding, 8px) 0 0 calc(2 * var(--app-navigation-padding, 8px) + 44px);
+		margin-inline: calc(2 * var(--app-navigation-padding, 8px) + 44px) var(--app-navigation-padding, 8px);
```

Comment:
I understand that previously, `margin-top` and `margin-left` were set, and now that `margin-inline-start` `margin-inline-end` are set, which are equivalent of `margin-left` and `margin-right`. Am I right? If so, don't we need to keep the `margin-top`?

Reply:
I am not sure about the top margin.

Keeping the top margin is ok but it is not aligned with the toggle button. 

![image](https://github.com/user-attachments/assets/d953458c-72be-44e3-8bb2-87e2d088bbc0)

Setting the top margin to zero aligns the title with the toggle button. See:
![image](https://github.com/user-attachments/assets/2668b290-7681-4de4-9ace-d65c7ed8105f)

Probably the designers are looking for comfortable margins around app content more than alignment with external elements. It guess it is safe to maintain the top margin.

Reply:
Your second screenshot looks good indeed. Let's keep it that way :).